tests/unittests/OperatorTest.cpp

```diff
@@ -1842,6 +1842,62 @@ TEST_P(InterpAndCPU, IntConcat) {
   }
 }
 
+TEST_P(Operator, FC) {
+  auto *input =
```

Comment:
We have IntFC in OperatorTest, which compares both float and quantized FC. Not sure if it is necessary to test this one again.

Reply:
Agreed, let's remove this, then.